--- Crypt-Blowfish-2.12/Blowfish.xs~	2000-11-30 07:25:21.000000000 +0100
+++ Crypt-Blowfish-2.12/Blowfish.xs	2013-06-09 19:49:55.063654083 +0200
@@ -63,8 +63,7 @@
 			output = sv_newmortal();
 		output_len = 8;
 
-		if (!SvUPGRADE(output, SVt_PV))
-			croak("cannot use output argument as lvalue");
+		SvUPGRADE(output, SVt_PV);
 
 		/* blowfish_crypt_8bytes(input, SvGROW(output, 8), ks, dir); */
 		/* HP-UX (HP cc) fix below, thanks Addi! */
